provision/provision-site.sh

```diff
@@ -99,10 +124,17 @@ function vvv_provision_site_nginx_config() {
   sed -i "s#{vvv_site_name}#${SITE_NAME}#"  "${TMPFILE}"
   sed -i "s#{vvv_hosts}#${VVV_HOSTS}#"  "${TMPFILE}"
 
+  # if nginx upstream is not 'php' then it has been changed in config.yml
   if [ 'php' != "${NGINX_UPSTREAM}" ] && [ ! -f "/etc/nginx/upstreams/${NGINX_UPSTREAM}.conf" ]; then
     vvv_error " * Upstream value '${NGINX_UPSTREAM}' doesn't match a valid upstream. Defaulting to 'php'.${CRESET}"
     NGINX_UPSTREAM='php'
   fi
+
+  # if php: is configured, set the upstream to match
+  if [ "${DEFAULTPHP}" != "${VVV_DEFAULTPHP}" ]; then
+    NGINX_UPSTREAM="php${DEFAULTPHP}"
+    NGINX_UPSTREAM=$(echo $NGINX_UPSTREAM | tr --delete .)
+  fi
```

Comment:
this should happen beforehand otherwise I could do the following:

```yaml
    php: 74.0
```
And Nginx would fail to load as there is no PHP v74.0

Reply:
ops I switched the lines